aireview/src/aireview/domain/services/code_analyzer.py

```diff
@@ -16,10 +17,9 @@ def analyze_code(self, pull_request: PullRequest) -> Review:
         # self._ai_agent.initialize(f"Creating context for review of pull request {pull_request.number}")
         for index, file in enumerate(pull_request.files):
```

Comment:
**style**: Removed debug print statements which improves code cleanliness in production environment